tapir/coop/migrations/0031_remove_createpaymentlogentry_end_date.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.15 on 2022-09-10 09:25
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('coop', '0030_createpaymentlogentry'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='createpaymentlogentry',
+            name='end_date',
+        ),
+    ]
```

Comment:
Could you merge that migration file with the previous one?
 - delete both files
 - call `makemigrations`
 - reset DB and test